subscriber_notifee.go

```diff
@@ -232,10 +184,6 @@ func (nn *subscriberNotifee) Disconnected(n network.Network, v network.Conn) {
 		return
 	}
 
-	dht.peerDisconnected(dht.ctx, p)
-
-	fixLowPeers(dht)
```

Comment:
Why did we remove `fixLowPeers` from here?

Reply:
Because a disconnection dosen't really do anything to the Routing Table.

Reply:
makes sense to me. If we want to handle routing table removal events or use a periodic goroutine to run `fixLowPeers` we can do that in another PR.